### ISSUE TITLE
[UX] How display a large database schema ?

### ISSUE THREAD

**Issue by mdartic** [locokit/locokit#37]
In the FédéSCoT instance, @bruhnild create a schema with tables and a lot of fields :

![image](https://user-images.githubusercontent.com/2027964/149171340-15aaac3d-4b4a-41ab-b179-844f4801ad26.png)

I find this schema hard to read, and the user need to zoom a lot before knowing on which table he's clicking.

Maybe we could search a way to display schema only with table names instead of table + field names ?
This could be a display option ?

What do you think @bruhnild @DarkVisenya @jcougnaud ?

**Comment by jcougnaud** [comment]
I agree with you, it's a good idea to display a lighter version of this schema.

I also think that displaying only the table names can be a good solution. We could have a switch button to switch from one mode to the other.

Another mode could be to display only the first n columns (the other columns would not be displayed at all or via a scroll bar, however, in this last case, I do not know if an overflow option is available with svg).
